CHANGELOG.md

```diff
@@ -1,35 +1,25 @@
 # Aries Cloud Agent Python Changelog
 
-## 1.0.1rc1
+## 1.0.1
 
-### October 1, 2024
+### October 8, 2024
 
-Release 1.0.1 _might_ be the last release of ACA-Py from the Hyperledger
-organization before the repository moves to the [OpenWallet Foundation] (OWF). As
-noted in the repository [README.md](README.md), the ACA-Py project will soon be
-moving soon to the OWF's GitHub organization as the [new "acapy"
-project](https://github.com/openwallet-foundation/project-proposals/pull/42).
+Release 1.0.1 will be the last release of ACA-Py from the Hyperledger organization before the repository moves to the [OpenWallet Foundation] (OWF).Soon after this release, the ACA-Py project and this repository will soon be moving soon to the OWF's GitHub organization as the [new "acapy" project](https://github.com/openwallet-foundation/project-proposals/blob/main/projects/aca-py.md).
```

Comment:
Too many "soon"s :-)

Suggested change from:

> (OWF).Soon after this release, the ACA-Py project and this repository will soon be moving soon to ...

to:
> (OWF). Soon after this release, the ACA-Py project and this repository will be moving to ...

Reply:
Too funny.  Comes from taking multiple passes at a line without re-reading it at the end.  Doh!!

Fixed.